python/cog/director/director.py

```diff
@@ -371,15 +362,15 @@ def _record_failure(self):
                 "saw too many failures in a row", failure_count=self._failure_count
             )
 
-    def _record_success(self):
+    def _record_success(self) -> None:
         self._failure_count = 0
 
-    def _abort(self, message=None, **kwds):
+    def _abort(self, message: str = None, **kwds: Any) -> None:
```

Comment:
I'm not quite sure why `message` was optional before but I don't think it needs to be or should be.
```suggestion
    def _abort(self, message: str, **kwds: Any) -> None:
```